versions.lock

```diff
@@ -22,7 +22,7 @@ com.google.guava:guava:28.1-jre (15 constraints: 8ef6e5ed)
 com.google.guava:listenablefuture:9999.0-empty-to-avoid-conflict-with-guava (1 constraints: bd17c918)
 com.google.j2objc:j2objc-annotations:1.3 (1 constraints: b809eda0)
 com.jcraft:jzlib:1.1.3 (1 constraints: 0705f635)
-com.netflix.concurrency-limits:concurrency-limits-core:0.2.2 (2 constraints: 3b148789)
+com.netflix.concurrency-limits:concurrency-limits-core:0.2.2 (1 constraints: 0605f335)
```

Comment:
@carterkozak didn't your re-implementation kill this dependency?

Reply:
Only for dialogue, I didn’t replace it from cjr-okttp.